app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        current_date = datetime.date.today()
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
+        if visitor["vaccine"]["expiration_date"] < current_date:
+            raise OutdatedVaccineError("OutdatedVaccineError")

```

Comment:
The error message 'OutdatedVaccineError' is not descriptive. As per the checklist, provide a more descriptive error message. You could use something like 'Visitor's vaccine is outdated.'